skbeam/core/accumulators/binned_statistic.py

```diff
@@ -379,43 +398,64 @@ def __init__(self, x, y, statistic='mean',
             bins = [xedges, yedges]
 
         super(BinnedStatistic2D, self).__init__([x, y], statistic=statistic,
-                                                bins=bins, range=range)
+                                                bins=bins, range=range,
+                                                mask=mask)
 
     def __call__(self, values):
         return super(BinnedStatistic2D, self).__call__(values)
 
 
+def get_r_phi(shape, origin):
```

Comment:
I would use utils.radial_grid and utils.angle_grid rather than add a new function. Drawback is you need two function calls. 
Another option is integrate this into utils somehow. Perhaps create a angle_radial_grid? As a placeholder, it could just call the two functions for now and eventually be changed into something more optimized.